tests/utils.js

```diff
@@ -845,4 +890,74 @@ module.exports = {
 
   getSettings: () => module.exports.getDoc('settings').then(settings => settings.settings),
 
+  prepServices: async () => {
+    if (constants.IS_TRAVIS) {
+      console.log('On travis, waiting for horti to first boot api');
+      // Travis' horti will be installing and then deploying api and sentinel, and those logs are
+      // getting pushed into horti.log Once horti has bootstrapped we want to restart everything so
+      // that the service processes get restarted with their logs separated and pointing to the
+      // correct logs for testing
+      await listenForApi();
+      console.log('Horti booted API, rebooting under our logging structure');
+      await rpn.post('http://localhost:31337/all/restart');
+    } else {
+      // Locally we just need to start them and can do so straight away
+      await rpn.post('http://localhost:31337/all/start');
+    }
+
+    await listenForApi();
+    const config = await browser.getProcessedConfig();
+    if (config.suite && config.suite === 'web'){
+      await runAndLog('Settings setup', setupSettings);
+    }
+    await runAndLog('User contact doc setup', setUserContactDoc);
+  },
```

Comment:
I think we shouldn't inline these large function directly in the export. It makes browsing the file a lot harder. Please make these separate `consts` and export the variables. 
This applies to prepServices, login (which we should also rename to something more specific like "browserLogin"),  setupUser and saveBrowserLogs